src/containers/Challenge.jsx

```diff
@@ -40,12 +40,24 @@ class ChallengeTabsClass extends React.Component {
 		const dlSize = "83 GB";
 		const scoreCategories = ["Score 1", "Score 2", "Score 3", "Score 4", "Score 5", "Score 6"];
 		if (this.state.active === "about") {
-			content = <About content={aboutContent} key="about" />;
+			content = <About color={challengeColor} content={aboutContent} key="about" />;
```

Comment:
This could be a function, if desired, rather than defined inline. If a lot more, can be cleaner.